### PR TITLE
PR 16150 use Symbol(x...) and deprecate symbol(x...)

### PR DESCRIPTION
**Pull request by jeffreysarnoff-dev**
picking up detached github branch for the automated checking (from [#16130](https://github.com/JuliaLang/julia/pull/16130#issuecomment-216049088))

### CODE REVIEW

File: base/deprecated.jl [hunk 1]
```diff
@@ -1082,6 +1082,9 @@ end
 @deprecate sprand{T}(n::Integer, density::AbstractFloat, ::Type{T}) sprand(T, n, density)
 @deprecate sprand{T}(r::AbstractRNG, n::Integer, density::AbstractFloat, ::Type{T}) sprand(r, T, n, density)
 
+# #16130
+@deprecated symbol Symbol
```
Comment (tkelman) [inline base/deprecated.jl]: `@deprecate`, not `@deprecated`

File: base/inference.jl [hunk 2]
```diff
@@ -1172,7 +1172,7 @@ function abstract_eval_gensym(s::GenSym, sv::InferenceState)
     return typ
 end
 
-function abstract_eval_symbol(s::Symbol, vtypes, sv::InferenceState)
+function abstract_eval_Symbol(s::Symbol, vtypes, sv::InferenceState)
```
Comment (tkelman) [inline base/inference.jl]: don't rename this

**Comment by yuyichao** [comment]
Could you please try harder to force push to the branch you are working on?
A new PR every time you update something is starting to be annoy...

You can practice on a different branch with possibly a PR on your fork if you are not sure what you are doing.

**Comment by tkelman** [comment]
All the past PR's now say "unknown repository" which I think can only happen if you delete your fork. That's really not necessary. The persistence is appreciated, but please ask for help with git rather than making more of a mess with multiple PR's.

**Comment by jeffreysarnoff-dev** [comment]
throwing in the towel

**Comment by tkelman** [comment]
We're a couple lines away this time. One of us can take over and finish the final few lines, though that would be a sixth PR for this. Unless you'd like to give one of us collaborator access to your fork and we could reopen and finish this for you?

**Comment by hayd** [comment]
I have fixed up a branch. Let me open a PR.

**Comment by hayd** [comment]
It looks like all of the above comments were already addressed.

**Pull request closed without merging**